src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.State.cs

```diff
@@ -59,7 +61,9 @@ private class State
                 public virtual bool? HasSuccessfullyLoaded => null;
 
                 /// <summary>
-                /// The final compilation if available, otherwise <see langword="null"/>.
+                /// The final compilation is potentially available, otherwise <see langword="null"/>.
+                /// The value is an <see cref="Optional{Compilation}"/> to represent the
+                /// possibility of the compilation already having been garabage collected.
```

Comment:
can you explicitly doc if SGs will or will not have run on this?

Reply:
Renamed as well.